YIF.Core.Domain/ApiModels/Validators/DirectionPostApiModelValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentValidation;
+using System.Linq;
+using System.Resources;
+using YIF.Core.Data;
+using YIF.Core.Domain.ApiModels.RequestApiModels;
+
+namespace YIF.Core.Domain.ApiModels.Validators
+{
+    public class DirectionPostApiModelValidator : AbstractValidator<DirectionPostApiModel>
+    {
+        private readonly EFDbContext _context;
+        private readonly ResourceManager _resourceManager;
+        public DirectionPostApiModelValidator(EFDbContext context, ResourceManager resourceManager)
+        {
+            _resourceManager = resourceManager;
+            _context = context;
+
```

Comment:
you should add here CascadeMode.Stop so when any field is invalid, validator throws exception and doesn't check further rules